tools/sggolangcilint/tools.go

```diff
@@ -33,15 +33,32 @@ func defaultConfigPath() string {
 }
 
 func CommandInDirectory(ctx context.Context, directory string, args ...string) *exec.Cmd {
-	configPath := filepath.Join(directory, ".golangci.yml")
-	if _, err := os.Lstat(configPath); errors.Is(err, os.ErrNotExist) {
-		configPath = defaultConfigPath()
+	skipConfig := false
```

Comment:
var skipConfig bool

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
I updated the code, which is now using `:=` consistently.

Reply:
Ok, I thought you meant to use some consistent declaration style... 😄 
I made the skipConfig into a var.